packages/angular_devkit/build_optimizer/BUILD.bazel

```diff
@@ -22,9 +17,6 @@ ts_library(
     srcs = glob(
         include = ["src/**/*.ts"],
         exclude = [
-            # TODO(@filipesilva): shouldn't need to exclude the cli files but can't exclude them
-            # from jasmine_node_test.
-            "src/**/cli.ts",
```

Comment:
That is an odd one to have been excluded. Dates back to 3 years ago https://github.com/angular/angular-cli/commit/c03516a81e72f05e1bb86fcd7f814cf07189c43d.

Was that `cli.ts` files present in the legacy built package?

Reply:
Ahh yes. It is there https://unpkg.com/browse/@angular-devkit/build-optimizer@0.1300.4/src/build-optimizer/

Reply:
Yep, any additions or removals I have in these rules is to replicate what's in the legacy built packages.